packages/ra-core/src/dataProvider/useMutation.spec.tsx

```diff
@@ -128,4 +134,236 @@ describe('useMutation', () => {
         await waitForDomChange({ container: testElement });
         expect(testElement.textContent).toEqual('provider error');
     });
+
+    it('supports declarative onSuccess side effects', async () => {
```

Comment:
Technically, you're testing `Mutation` and not `useMutation` here. I'd move the tests to avoid confusion.

Reply:
The whole file is testing `Mutation`. Should I rename it ?